src/zampy/datasets/land_cover.py

```diff
@@ -141,7 +141,7 @@ def load(
         grid = xarray_regrid.create_regridding_dataset(
             utils.make_grid(spatial_bounds, resolution)
         )
-        ds = ds.regrid.most_common(grid, time_dim="time", max_mem=1e9)
+        ds = ds.regrid.most_common(grid)
```

Comment:
It should work a lot better now :)

Reply:
hmmm... not yet, `values` is a required argument now :smile:

Reply:
Yeah but the unique values should be encoded in the land cover dataset's attributes:

`da.attrs["flag_values"]`

Reply:
I found also this issue https://github.com/xarray-contrib/xarray-regrid/issues/52

Reply:
That issue is fixed now. But there isn't a new release yet.